src/webview/linux/mod.rs

```diff
@@ -135,9 +135,6 @@ impl InnerWebView {
       settings.set_enable_offline_web_application_cache(true);
       settings.set_enable_page_cache(true);
 
-      // Enable Smooth scrooling
```

Comment:
Can we make this optional?

Reply:
Only Linux has this. I haven't found other platforms also have this option.